.github/pytorch-probot.yml

```diff
@@ -15,6 +15,7 @@ ciflow_push_tags:
 - ciflow/trunk
 - ciflow/unstable
 - ciflow/xpu
+- ciflow/aarch64
```

Comment:
This should fix linter, though it makes it really hard how to test what is going on
```suggestion
```